src/Serilog/Serilog.csproj

```diff
@@ -52,4 +56,8 @@
     <PackageReference Include="System.Collections.NonGeneric" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="System.Collections.NonGeneric" Version="4.3.0" />
```

Comment:
I just started playing with Serilog again and noticed it pulling in a bunch of unneeded packages. This is the root cause. Which collection is being used and what isn't it generic? 😄

Reply:
😄 it's this one: https://github.com/serilog/serilog/blob/dev/src/Serilog/Core/Pipeline/MessageTemplateCache.cs#L32

IIRC, `Hashtable`'s contract supports concurrent/unsynchronized reads in the presence of a single writer, while `Dictionary<K,V>` dropped this.

Reply:
> PS: Everything versioned 4.3.0 are packages that existed pre-netstandard 2.0. If they don't have a newer version then it was included in the standard.